### PR TITLE
3674: adding network adapter to metadata

### PR DESCRIPTION
**Pull request by tamanna-k**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

File: svc-api/handle/handlers.go [hunk 1]
```diff
@@ -535,6 +535,12 @@ func GetMetadata(ctx iris.Context) {
 					models.Include{Namespace: "FabricCollection"},
 				},
 			},
+			models.Reference{URI: "http://redfish.dmtf.org/schemas/v1/NetworkAdapter_v1.xml",
+				TopInclude: []models.Include{
+					models.Include{Namespace: "NetworkAdapter"},
+					models.Include{Namespace: "NetworkAdapter.v1_5_0"},
+				},
+			},
 		},
```
Comment (Bharath-KKB) [inline svc-api/handle/handlers.go:547]: include Namespace="NetworkAdapterCollection" as well

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
add Namespace="NetworkAdapterCollection"

File: svc-api/handle/handlers.go (hunk 1 above)
Comment (tamanna-k) [reply svc-api/handle/handlers.go:547]: 'NetworkAdapterCollection' is already present (line 443)

Comment (Bharath-KKB) [reply svc-api/handle/handlers.go:547]: OK add Namespace="NetworkAdapter.v1_2_0" Namespace="NetworkAdapter.v1_3_0" Namespace="NetworkAdapter.v1_4_0" to NetworkAdapter

**Review by Bharath-KKB — APPROVED** [review]
Address comments

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
address comments

File: svc-api/handle/handlers.go (hunk 1 above)
Comment (tamanna-k) [reply svc-api/handle/handlers.go:547]: done

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**